src/js/tether.js

```diff
@@ -102,39 +101,26 @@ class TetherClass extends Evented {
 
     this.setOptions(options, false);
 
-    TetherBase.modules.forEach((module) => {
-      if (!isUndefined(module.initialize)) {
-        module.initialize.call(this);
-      }
-    });
+    Marker.initialize.call(this);
 
     this.position();
   }
 
-  getClass(key = '') {
```

Comment:
Can we do `this.getClass = getClass.bind(this)` in the constructor and change all the `getClass.call(this, 'foo')` back to `this.getClass('foo')`?

Reply:
I don’t understand what that infers since this is changed to an imported util that you suggest to obfuscate to the class?

Reply:
I'm not sure I am following